### ISSUE TITLE
Glossary Content Type

### ISSUE THREAD

**Issue by alex-418** [BCcampus/pressbooks#31]
Create content type called "Glossary", location and number of menu items TBD.

**Comment by bdolor** [comment]
mostly done via https://github.com/BCcampus/pressbooks/commit/0b20e972eb6cac3cda9874f6b2a1bf9418cee1c0 

leaving open for now to account for inevitable changes

**Comment by bdolor** [comment]
this is complete. @josieg will need you to confirm that things are working as expected. Some things to note: 

- you may see a warning in the editor/dashboard about a glossary script not loading - please ignore, the branch we're working on hasn't been merged with the branch that fixes that. 
- to validate, just be sure that you can add glossary terms, and glossary types from the dashboard and that things are working in a manner that you would expect.

**Comment by josieg** [comment]
Looks good. I was able to add a number of glossary terms and put them into different types that I created. 

This may be beyond the scope of this particular issue but here are a few notes:

- the "Add Glossary Term" label in the menu is misleading. Clicking on that option takes the user to where they can see and manage glossary terms. They have to click another button once on that page to actually add a term. Can we change it to "Glossary Terms"?
- on the "Glossary Terms" page, the terms are organized in the order that I added them, but I think it would be preferable if they were alphabetized so they are easy to find.

**Comment by bdolor** [comment]
I agree the label is misleading and that it would be more accurate with 'Glossary Terms'. 

The alphabetizing of words happens on the display side of things. If you are referring to wanting it alphabetized in the admin dashboard, and the problem to be solved is finding them I'd recommend keeping it the way is for the following reasons: 
- If you have enough terms that the admin page starts to paginate, and a glossary term is added that starts with 'z', a user is then forced to click through many admin pages to get to the term they just added. (more likely to want to edit an item that has recently been added)
- the way it's currently done is in line with default WP behaviour (some would therefore come to expect it?).
- there is a search bar in the top right to aid in locating a particular glossary term.

**Comment by josieg** [comment]
Sounds good!

**Comment by bdolor** [comment]
label updated via https://github.com/BCcampus/pressbooks/commit/01662ec77dedf5b2de8b9a5b6ceef01b2ccc9111
